lib/Mojo/WebSocketProxy/Backend/JobAsync.pm

```diff
@@ -66,7 +66,7 @@ sub new {
     # We'd like to provide some flexibility for people trying to integrate this into
     # other systems, so any combination of Job::Async::Client, Job::Async and/or IO::Async::Loop
     # instance can be provided here.
-    $self->{client} //= do {
+    unless ($self->{client}) {
```

Comment:
@mat-binary I dont see the need of changing this part

Reply:
Yes. It is a leftover of an experiment. I'll revert it.